econplayground/main/models.py

```diff
@@ -360,6 +360,10 @@ class Meta(OrderedModel.Meta):
     expression_2 = models.TextField(default='', blank=True)
     expression_3 = models.TextField(default='', blank=True)
 
+    # A graph may display different grid types
+    major_grid = models.PositiveIntegerField(default=0)
```

Comment:
Can we call these variables `major_grid_type` and `minor_grid_type`, as that better explains what their purpose is. If it's called `major_grid`, I would expect this to be a representation of the grid itself.

Reply:
Done